Adjust integration name and branding

nikhilguptanetnologyio opened the pull request:
### What does this PR do?

Fixes potential branding inaccuracies and adjusts integration name accordingly.

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

nikhilguptanetnologyio commented on nn_sdwan/manifest.json:
```diff
@@ -1,5 +1,5 @@
 {
-  "display_name": "Netnology SD-WAN",
+  "display_name": "Netnology Cisco SD-WAN",
```
Done!

nikhilguptanetnologyio commented on .codecov.yml:
```diff
@@ -203,14 +211,6 @@ coverage:
         target: 75
         flags:
         - pulsar
-      PureFA:
```
Correct me if I'm wrong, but the latest commit on master at the moment looks to be 33ebbd6d, which is what this PR works off of.

sarah-witt replied:
Hm, could you try reverting the change in codecov.yml and see if validation still pases?

sarah-witt commented:
<img width="220" alt="Screen Shot 2022-06-08 at 2 19 37 PM" src="https://user-images.githubusercontent.com/33498636/172688790-7fa316a1-011a-47b6-94e8-421ecfa06227.png">
@nikhilguptanetnologyio Everything looks good! Once you update `.codecov.yml` this should be good to go!

nikhilguptanetnologyio commented:
@sarah-witt thanks, I reverted `.codecov.yml` back to 33ebbd6 and all the checks still pass. Since we changed the readme a bit, [our integration's docs page](https://docs.datadoghq.com/integrations/nn_sdwan/) will need to be updated as well. Will the link to that change?

sarah-witt commented:
@nikhilguptanetnologyio thanks! Yes, the docs page will be updated once this change is merged. The link will stay the same though: https://docs.datadoghq.com/integrations/nn_sdwan/

sarah-witt commented:
@nikhilguptanetnologyio Let me know when you are ready for us to merge this!

nikhilguptanetnologyio commented:
@sarah-witt @anawishnoff We're good to go, you can merge this. Thank you!
